### ISSUE TITLE
Sync-NavContainerApp Mode ForceSync

### ISSUE THREAD

**Issue by MatthewMcCarthy7** [microsoft/navcontainerhelper#1156]
**Describe the issue**
The Mode ForceSync Option is not longer Available in the NavContainerHelp Sync-NavContainerApp

**Full output of scripts**
```
NavContainerHelper is version 0.7.0.19
NavContainerHelper is running as administrator
...
```

**Screenshots**
![image](https://user-images.githubusercontent.com/32099690/88322568-c006e880-ccd5-11ea-8bf4-9485020aba56.png)

**Comment by freddydk** [comment]
Was it ever?
I cannot find evidence in the source that it has ever been there.
That doesn't mean we shouldn't add it:-)

It is in Publish-NavContainerApp - where you can say sync and forcesync.

**Comment by MatthewMcCarthy7** [comment]
I thought it was in there and used it before but could have been confused with the On-Prem Sync-NavApp.  That would be nice if we could add it to the Sync-NavContainerApp.

**Comment by freddydk** [comment]
Shipped in 0.7.0.20
